src/ibex_bluesky_core/callbacks/fitting/livefit_logger.py

```
@@ -64,9 +71,14 @@ def start(self, doc: RunStart) -> None:
             doc (RunStart): The start bluesky document.
 
         """
+        datetime_obj = datetime.fromtimestamp(doc[TIME])
+        title_format_datetime = datetime_obj.astimezone(ZoneInfo("Europe/London")).strftime(
+            "%Y-%m-%d_%H-%M-%S"
+        )
         self.output_dir.mkdir(parents=True, exist_ok=True)
         self.current_start_document = doc[UID]
-        self.filename = self.output_dir / f"{self.current_start_document}{self.postfix}.csv"
+        file = f"{INSTRUMENT}_{self.x}_{self.y}_{title_format_datetime}Z{self.postfix}.csv"
+        self.filename = self.output_dir / f"{doc.get('rb_number', None)}" / file
```

Is rb_number an actual key of RunStart?

---

Should probably explicitly handle the case where we get `None` though (e.g. throw an obvious error rather than putting data into `\\some\network\path\None\some_data`)